docs/mango.md

```diff
@@ -116,3 +116,49 @@ Content-Type: application/json
 - The sort field must match an existing index
 - It is possible to sort in reverse direction `sort:[{"calendar":"desc"}, {"date": "desc"}]` but **all fields** must be sorted in same direction.
 - `use_index` is optional but recommended.
+
+
+## Pagination cookbook
+
+Pagination of mango query should be handled by the client.
+The stack will always limit query results to maximum 100 docs.
+
+If the limit cause some docs to not be returned, the response will have a
+limit & next values.
+
+```json
+{
+    "limit": 100,
+    "next": true,
+    "docs": [ "... first hundred docs ..."],
+}
+```
+
+If there is the number of docs is lower or equal to the limit, these fields are not present
+
+```json
+{
+    "docs": [ "... less than a hundred docs ..."],
+}
+```
```

Comment:
Maybe I'm wrong, but I would better see an api where we always put the `next` and `limit` field (even if they are `false` and the default value) to be more explicit ?

Also IMHO, it is better than having to handle `undefined`s.